From a53f3fd64a0f36ec9bc1307d7e66cdc2dee4aeaa Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Tue, 4 Aug 2015 10:14:00 +0200
Subject: [PATCH] poco: add the staging path to search path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add the mysql headers and client libraries to the search path of the
preprocessor and the linker. The $MYSQL_LIBDIR / $MYSQL_INCIDR variables
must be set from the make command line.

[Peter: Remove host dirs, add MYSQL_INCDIR]
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
[Jörg: Update to version 1.6.1 from github]
Signed-off-by: Jörg Krause <joerg.krause@embedded.rocks>
[Julien: Rebased on version 1.10.1]
Signed-off-by: Julien Olivain <juju@cotds.org>
---
 Data/MySQL/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Data/MySQL/Makefile b/Data/MySQL/Makefile
index 64a7d3f82..955fee2d4 100644
--- a/Data/MySQL/Makefile
+++ b/Data/MySQL/Makefile
@@ -6,8 +6,8 @@
 
 include $(POCO_BASE)/build/rules/global
 
-SYSLIBS += -L/usr/local/lib -L/usr/local/lib$(LIB64SUFFIX)/mysql -L/usr/lib$(LIB64SUFFIX)/mysql -L/usr/mysql/lib$(LIB64SUFFIX) -L/usr/mysql/lib$(LIB64SUFFIX)/mysql -L/usr/local/mysql/lib$(LIB64SUFFIX) -lmysqlclient
-INCLUDE += -I/usr/local/include/mysql/ -I/usr/include/mysql/ -I/usr/mysql/include/mysql -I/usr/local/mysql/include
+SYSLIBS += -L$(MYSQL_LIBDIR) -lmysqlclient
+INCLUDE += -I$(MYSQL_INCDIR)
 SYSFLAGS += -DTHREADSAFE -DNO_TCL
 
 objects = Binder Extractor SessionImpl Connector \
-- 
2.26.2

